xml_inscriptions/bib_only/MA.Bos.MFA.G.01.Res.08.34b.xml

```diff
@@ -35,18 +35,86 @@
+               <physDesc>
+                  
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#sculpture">
+                     <supportDesc ana="#stone.marble">
+                        <support>
+                           <p> (attic) marble votive plaque</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>15</height>
+                              <width>12.4</width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#broken">
+                           <p>unevenly carved and broken at upper left. Back roughly finished.
+                              Slight yellow patina.</p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="2">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p>inscribed circle above text </p>
+                     </decoNote>
+                     <decoNote>
+                        <p>inscribed star below text </p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               
+               <!--  *************************************  <history>  *********************************  -->
+               
+               <history>
+                  
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#unspecified" notBefore="0099" notAfter="0300">2nd–3rd century
+                        A.D.</date>
+                     <placeName ref="europe.greece.athens"/>
+                  </origin>
+                  <provenance type="transferred" subtype="gifted">
+                     <p> In <date when="1908">1908</date>, gift of Fiske Warren and Edward Perry Warren to the MFA. (Accession date: January 1, 1910) </p>
```

Comment:
Note that we can add `<persName>` and `<orgName>` here.

Reply:
Yes, please be sure to add the correct name tags (pers. org. place) where needed